test/test_python_binary_dist.py

```diff
@@ -72,6 +72,13 @@ def get_allowed_deps():
                 'libtinfo.so.5',
                 ])
         else:
+            # Raspbian is a special case... If this file exists, we assume the
+            # distro passes the more stringent checks during the build phase.
```

Comment:
just a small note.
This comment is actually related to the code inside the `if os.path.isfile("/etc/rpi-issue"):` block so it should be indented at the same level as the `allowed_deps.extend(['libcofi_rpi.so',` code.